CODEOWNERS

```diff
@@ -312,8 +312,8 @@
 /bundles/org.openhab.binding.russound/ @openhab/add-ons-maintainers
 /bundles/org.openhab.binding.sagercaster/ @clinique
 /bundles/org.openhab.binding.saicismart/ @tisoft @dougculnane
+/bundles/org.openhab.binding.samsungtv/ @NickWaterton
```

Comment:
The conflict was with salus, it was added to the exact same line. after you created the PR it was merged.
Now you fixed the conflict, but samsung is now above salus and it should be below (alphabetically) 
Sorry this small change is taking so much time.

Reply:
Ok, moved it.